LINUX/default-config.mak.in_

```diff
@@ -136,3 +136,23 @@ virtio_net.c@force	:= 1
 endef
 
 $(foreach d,$(filter virtio_net.c,$(E_DRIVERS)),$(eval $(call virtio_net)))
+
+# need to sub release string on the line here
+bnxt_en_ver := 1.10.2
+
+define bnxt_driver
+$(1)@fetch	:= test -e @SRCDIR@/../bnxt_en-$(bnxt_en_ver)-$(2).tar.gz && cp @SRCDIR@/../bnxt_en-$(bnxt_en_ver)-$(2).tar.gz -P @SRCDIR@/ext-drivers
```

Comment:
If these sources cannot be downloaded automatically, I think we can just let the fetch fail with a message that points the user to the proper url, i.e., something like
```
$(1)fetch := false && please download bnxt_en-$(bnxt_en_ver)-$(2).tar.gz from https://whatever
```
The `configure` message already tells the user to put the file in `@SRCDIR@/ext-drivers`.

Reply:
I'm working with my management to figure out a way of getting a downloadable URL